lib/vsc/administration/user.py

```diff
@@ -305,16 +314,23 @@ def _create_user_dir(self, grouping_f, path_f, storage_name):
         @type grouping: function that yields the grouping path for the location.
         @type path: function that yields the actual path for the location.
         """
+        try:
+            storage = self.institute_storage[storage_name]
+        except KeyError:
+            logging.exception("Trying to access non-existent institute storage: %s", storage_name)
```

Comment:
`Trying -> Failed

Reply:
done in https://github.com/hpcugent/vsc-administration/pull/134/commits/d9d2f11e38cfec377807ac56339b0498d1b1400d